specs/phase1/validator.md

```diff
@@ -281,9 +281,9 @@ def get_shard_transition_fields(
     shard: Shard,
     shard_blocks: Sequence[SignedShardBlock],
 ) -> Tuple[Sequence[uint64], Sequence[Root], Sequence[ShardState]]:
-    shard_states = []
-    shard_data_roots = []
-    shard_block_lengths = []
+    shard_block_lengths = []  # type: PyList[uint64]
+    shard_data_roots = []  # type: PyList[Root]
+    shard_states = []  # type: PyList[ShardState]
```

Comment:
`PyList` was not meant to make it into the spec. More type safety is nice, but I am hesitant to introduce this.

Reply:
I was hesitant too, and intentionally used type comments instead of `shard_block_lengths: PyList[uint64] = []`. 😅 But I understand your concerns.

A "PyList is okay" argument is that we already have Python `Set`, `Dict`, and `Tuple` in the specs.

Would renaming `PyList` to `Array` be better?

Reply:
`PyList` is okay then I guess. And sorry for nitpicking, it's just a style thing that we tried to avoid previously, but typing is more important.